configs/semi/faster_rcnn/semi_base_faster_rcnn_r50_caffe_fpn_180k_partial_coco.py

```diff
@@ -0,0 +1,49 @@
+_base_ = [
+    '../../_base_/models/faster_rcnn_r50_fpn.py',
+    '../../_base_/default_runtime.py', '../semi_partial_coco_detection.py'
+]
+
+model_wrapper = dict(
```

Comment:
`model_wrapper` should be distributed model wrapper such as MMDDP, not a detector.